tests/playwright-test/reporter.spec.ts

```diff
@@ -735,6 +735,36 @@ test('step attachments are referentially equal to result attachments', async ({
   ]);
 });
 
+test('step.attach attachments are referentially equal to result attachments', async ({ runInlineTest }) => {
```

Comment:
I wonder how does this test check the referential equality 😄

Reply:
It doesn't, updated the title.

Reply:
It doesn't, updated the title.